### PR TITLE
fix(translations): update section naming

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix MANAGER-9273
| License          | BSD 3-Clause

## Description

Rename sections

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "telephony_breadcrumb": "Téléphonie",
+  "telephony_breadcrumb": "VOIP",
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/translations/Messages_fr_FR.json:2]: **question:** Shouldn't be `VoIP` instead?

File: packages/manager/modules/cda/src/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "cda_title": "Ceph cluster"
+  "cda_title": "Cloud Disk Array"
```
Comment (antleblanc) [inline packages/manager/modules/cda/src/translations/Messages_fr_FR.json:2]: **question:** If content is not meant to be translated, couldn't we use it directly as a constant?

File: packages/manager/modules/email-domain/src/email/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "email_domain_email": "Emails"
+  "email_domain_email": "MXPlan"
```
Comment (antleblanc) [inline packages/manager/modules/email-domain/src/email/translations/Messages_fr_FR.json:2]: **question:** Shouldn't be `MX Plan` instead?

File: packages/manager/modules/emailpro/src/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
-  "email_pro_title": "Emails Pro",
+  "email_pro_title": "Email Pro",
```
Comment (antleblanc) [inline packages/manager/modules/emailpro/src/translations/Messages_fr_FR.json:2]:
```suggestion
  "email_pro_title": "E-mail Pro",
```

File: packages/manager/modules/iplb/src/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -1,3 +1,3 @@
 {
-  "iplb_title": "LoadBalancer"
+  "iplb_title": "OVHcloud Load Balancer"
```
Comment (antleblanc) [inline packages/manager/modules/iplb/src/translations/Messages_fr_FR.json:2]: **question:** If content is not meant to be translated, couldn't we use it directly as a constant?

File: packages/manager/modules/pci/src/components/project/users/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "pci_projects_project_users_title": "Utilisateurs",
+  "pci_projects_project_users_title": "Users & Roles",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/components/project/users/translations/Messages_fr_FR.json:2]: **question:** If content is not meant to be translated, couldn't we use it directly as a constant?

File: packages/manager/modules/pci/src/projects/project/billing/legacy/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "cpbc_billing_control": "Facturation",
+  "cpbc_billing_control": "Billing control",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/billing/legacy/translations/Messages_fr_FR.json:2]: **question:** If content is not meant to be translated, couldn't we use it directly as a constant?

**Review by antleblanc — COMMENTED** [review]
Looks like some content aren't meant to be translated.

Morever some english content are stored into french translation files.

Couldn't we handle them as static content/constant?

**Comment by cbourgois** [comment]
> Looks like some content aren't meant to be translated.
> 
> Morever some english content are stored into french translation files.
> 
> Couldn't we handle them as static content/constant?

Need more infos from requesters. Blocked for now.

**Review by frenautvh — APPROVED** [review]

**Review by Jisay — APPROVED** [review]

**Pull request merged**